src/rand.rs

```diff
@@ -198,13 +198,13 @@ mod sysrand {
     }
 }
 
-// Keep the `cfg` conditions in sync with the conditions in lib.rs.
 #[cfg(all(
     feature = "use_heap",
-    any(target_os = "redox", unix),
-    not(any(target_os = "macos", target_os = "ios")),
     not(all(target_os = "linux", not(feature = "dev_urandom_fallback"))),
-    not(any(target_os = "fuchsia")),
+    not(target_os = "macos"),
+    not(target_os = "ios"),
+    not(target_os = "fuchsia"),
+    not(target_os = "windows"),
```

Comment:
I am planning to switch this to a whitelist approach because I want to avoid using `/dev/urandom` whenever practical. So I think that while this may work right now, it will soon (in a matter of days) stop working.